src/legacy/core_plugins/kibana/migrations/migrations.js

```diff
@@ -60,6 +63,7 @@ function migrateIndexPattern(doc) {
 
 export const migrations = {
   dashboard: {
+    '6.7.2': migrateMatchAllQuery,
```

Comment:
Not sure that version '6.7.2' is a correct version, anyway this PR will be delivered only for versions >`7.8` so it's not so important